io.atom.electron.Sdk.json.in

```diff
@@ -0,0 +1,268 @@
+                    "path": "libnotify-disable-tests.patch"
+                }
+            ]
+        },
+        {
+            "name": "gvfs",
+            "cleanup": [ "/libexec/*", "/share/dbus-1/services/*", "/share/gvfs/mounts" ],
+            "config-opts": [ "--disable-hal", "--disable-gdu", "--disable-gcr", "--disable-obexftp",
+                             "--disable-avahi", "--disable-documentation", "--disable-admin" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gvfs/1.28/gvfs-1.28.0.tar.xz",
+                    "sha256": "cf72fc0adf0ca702ead5b3fab3c1fa46b09678eb7c1290de7e30bb7cbaf5f704"
+                }
+            ]
+        },
+        {
+            "name": "libpgp-error",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.24.tar.bz2",
+                    "sha256": "9268e1cc487de5e6e4460fca612a06e4f383072ac43ae90603e5e46783d3e540"
+                }
+            ]
+        },
+        {
+            "name": "libgcrypt",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.3.tar.bz2",
+                    "sha256": "ddac6111077d0a1612247587be238c5294dd0ee4d76dc7ba783cc55fb0337071"
+                }
+            ]
+        },
+        {
+            "name": "libgnome-keyring",
+            "config-opts": [ "--disable-static", "--disable-gtk-doc", "--disable-coverage",
+                             "--disable-introspection" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgnome-keyring/3.12/libgnome-keyring-3.12.0.tar.xz",
+                    "sha256": "c4c178fbb05f72acc484d22ddb0568f7532c409b0a13e06513ff54b91e947783"
+                }
+            ]
+        },
+        {
+            "name": "libdbusmenu",
+            "config-opts": [ "--with-gtk=2", "-disable-static", "--disable-tests", "--disable-gtk-doc",
+                             "--disable-doc", "--disable-dumper", "--enable-introspection=no", "--disable-vala" ],
+            "build-options" : {
+                "env": {
+                    "HAVE_VALGRIND_TRUE": "'#'",
+                    "HAVE_VALGRIND_FALSE": "''"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/libdbusmenu/12.10/12.10.2/+download/libdbusmenu-12.10.2.tar.gz",
+                    "sha256": "9d6ad4a0b918b342ad2ee9230cce8a095eb601cb0cee6ddc1122d0481f9d04c9"
+                },
+                {
+                    "type": "patch",
+                    "path": "libdbusmenu-remove-gtk-doc-init.patch"
```

Comment:
You can add this to avoid this patch: https://git.gnome.org/browse/gnome-doc-utils/
But if you don't, the patch should be renamed to mention `gnome-doc` instead of `gtk-doc`